pipeline/load.py

```diff
@@ -0,0 +1,291 @@
+"""This is the script to load plant data into the database"""
+from os import environ
+from datetime import datetime
+import logging
+import ast
+from typing import Optional
+import pandas as pd
+import psycopg2
+from psycopg2 import extensions
+from dotenv import load_dotenv
+
+load_dotenv()
+
+
+def config_log() -> None:
+    """
+    Terminal logs configuration.
+    """
+    logging.basicConfig(
+        format="{asctime} - {levelname} - {message}",
+        style="{",
+        datefmt="%Y-%m-%d %H:%M",
+        level=logging.INFO,
+    )
+
+
+def get_connection() -> extensions.connection:
+    """
+    Tries to connect to the RDS database.
+    Returns a connection if successful.
+    """
+
+    logging.info("Connecting to the database.")
+
+    try:
+        connection = psycopg2.connect(
+            host=environ["DB_HOST"],
+            port=environ["DB_PORT"],
+            user=environ["DB_USER"],
+            password=environ["DB_PASSWORD"],
+            database=environ["DB_NAME"]
+        )
+        return connection
+    except psycopg2.OperationalError:
+        logging.warning("The database %s doesn't exist", environ["DB_NAME"])
+        return None
+
+
+def get_cursor(connection: extensions.connection) -> Optional[extensions.cursor]:
+    """
+    Returns a the cursor for querying
+    the database from a connection.
+    """
+    try:
+        return connection.cursor()
+    except:
+        logging.warning("Unable to retrieve cursor for connection")
+        return None
+
+
+def get_id_from_country(country_name: str, cursor: extensions.cursor) -> int:
+    """
+    Retrieves the country_id of a 
+    given country from the database.
+    Returns -1 if it cannot be found.
+    """
```

Comment:
I think for doc strings we should just include what the function does - the type hints show what the function returns so I think we can take this out. It will also keep consistency with docstrings in other files.